### ISSUE TITLE
Comments at the end of the line are not processed correctly

### ISSUE THREAD

**Issue by igorkulman** [igorkulman/iOSLocalizationEditor#52]
Mentioned in #51 when a comment is at the end of the line, like

```
"Tr.trash_verb" = "löschen"; /* as a verb, i.e. to trash */
```

it does not get assigned to the correct string but to the next one.

**Comment by igorkulman** [comment]
@Tantalum73 any chance you can help with this as the author of the parser?

**Comment by Tantalum73** [comment]
I am happy to!
Currently the parser relies on the order of comment, key and value. 

I could imagine a setting where the user can specify the order. That may fix the problem. 

An automatic guess about the order may be a bit tricky. I haven’t founds hints that tell to which line the comments belong to (new lines are also out since a valid token may contain a new line).

**Comment by igorkulman** [comment]
Great, the help would be much appreciated. I only touched the parser once adding single line comments support in a hacky way which could be also improved. Thanks again.

As for the order guessing I think newlines might be a lead.

**Comment by igorkulman** [comment]
Fixed by #53
